etc/helm/pachyderm/values.yaml

```diff
@@ -215,6 +215,55 @@ loki-stack:
       tolerations: []
   grafana:
     enabled: false
+  promtail:
+    config:
+      lokiAddress: http://{{ .Release.Name }}-loki:3100/loki/api/v1/push
```

Comment:
I didn't realize we can resolve substitutions in values.yaml

Reply:
I stole it from https://github.com/grafana/helm-charts/blob/main/charts/loki-stack/values.yaml#L8